### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by f2prateek**
cc @calvinfo

### CODE REVIEW

**Comment by f2prateek** [comment]
not sure why travis didn't put the green check https://travis-ci.org/segmentio/analytics-python/builds/58803924

File: analytics/test/request.py [hunk 1]
```diff
@@ -15,15 +15,16 @@ def test_valid_request(self):
...
+        res = post('testsecret', batch=[{
             'userId': 'userId',
             'event': 'python event',
             'type': 'track'
         }])
+        self.assertEqual(res.status_code, 200)
 
     def test_invalid_request_error(self):
-        self.assertRaises(Exception, post, 'testsecret', batch='invalid')
+        self.assertRaises(Exception, post, 'testsecret', '[{]')
```
Comment (calvinfo) [inline analytics/test/request.py]: nice, dig it

File: analytics/test/request.py [hunk 2]
```diff
@@ -15,15 +15,16 @@ def test_valid_request(self):
         }])
         self.assertEqual(res.status_code, 200)
 
-    def test_invalid_write_key(self):
-        self.assertRaises(Exception,  post, 'invalid', batch=[{
+    def test_success_on_invalid_write_key(self):
```
Comment (calvinfo) [inline analytics/test/request.py]: looking at it now, I think we can probably get rid of this test, since it's basically the same case as above

**Comment by calvinfo** [comment]
boom dude, I think we can get rid of those two cases, but the invalid test lgtm! should be ready to merge at that point

File: analytics/test/request.py (hunk 2 above)
Comment (f2prateek) [inline analytics/test/request.py]: sgtm!

**Pull request merged**